javascript-typing-tutor/main.js

```diff
@@ -0,0 +1,16 @@
+var $span = document.querySelectorAll('span');
+document.addEventListener('keydown', keyHandler);
+var i = 0;
+
+function keyHandler(event) {
+  $span[i].className = 'underline';
+  if (i === ($span.length)) {
+    $span[i].className = 'right';
```

Comment:
Instead of setting the className, just put a `return` in here. If `i === $span.length`, the game is over.